packages/model/src/messages/mrtd/EMrtdDataSubmitMessage.ts

```diff
@@ -122,4 +123,53 @@ export class EMrtdDataSubmitMessage extends BaseMessage {
     }
     return newEmrtdData
   }
+
+  /**
+   * Converts a Machine Readable Travel Document (MRTD) date in the format `YYMMDD` to a complete
+   * `YYYYMMDD` date format, taking into account the current century.
+   *
+   * **Note:** This method is limited to interpreting dates based on the current year.
+   * It may not handle dates correctly for years beyond the range determined by the
+   * current century (e.g., for dates after 2050 when the current year is in the 21st century).
+   *
+   * @param {string} date - The MRTD date string in the format `YYMMDD`.
+   * @param {boolean} isExpirationDate - A boolean flag indicating whether the date is an expiration date.
+   * @returns {string} - The converted date in the format `YYYYMMDD`, or the original input
+   *                     if the input is not a valid `YYMMDD` date.
+   *
+   * @example
+   * // Current year: 2024
+   * convertMRTDDate("240101"); // Returns "20240101"
+   * convertMRTDDate("991231"); // Returns "19991231"
+   * convertMRTDDate("abcd12"); // Returns "abcd12" (invalid input)
+   */
+  public convertMRTDDate(date: string | null | undefined, isExpirationDate: boolean) {
```

Comment:
I think this can be an utility method in another file. And it is a great opportunity to introduce unit testing

Reply:
implemented